Packs/McAfeeNSM/Integrations/McAfeeNSMv2/McAfeeNSMv2_test.py

```diff
@@ -759,6 +760,7 @@ def test_list_domain_rule_objects_command(mocker, mcafeensmv2_client):
     response = util_load_json('test_data/commands_test_data.json').get('list_domain_rule_objects')
     expected_result = response.get('RuleObjDef')
     mocker.patch.object(mcafeensmv2_client, 'list_domain_rule_objects_request', return_value=response)
+    mocker.patch.object(McAfeeNSMv2, 'VERSION', 'V9x')
```

Comment:
As I suggested in our last conversation, you can test here for both versions (v9 and v10) using parameterize.
Relevant to this test and the next 3 tests.